### DISCUSSION TITLE
`Aave' keyword duplicated

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [solodit/feedback#22] [Bug Report] [answered]
![image](https://github.com/solodit/feedback/assets/33568293/999c0c9e-d662-4cb7-85f5-6f3acfb60bf2)

**Comment by wang1252** [comment, marked as answer]
Fixed.
Thank you for finding. @IAm0xAxe 

![image](https://github.com/solodit/feedback/assets/133619360/b21c8223-ce30-4f17-92d1-6300b7556995)

**Reply by IAm0xAxe** [reply]
You're welcome ^_^
